package.json

```diff
@@ -10,7 +10,11 @@
     "build": "tsc",
     "semantic-release": "semantic-release",
     "test": "jest src --coverage",
-    "test:e2e": "jest e2e",
+    "test:e2e": "jest e2e --runInBand",
```

Comment:
I think we ought to either make `test:e2e` call all the chained helper functions _or_ just toss `jest e2e --runInBand` onto `test:e2e:run`. That seems most-consistent with our other testing patterns in Pepr & Pexex.